.docker/entrypoint/spec/jekyll_commander_spec.rb

```diff
@@ -73,4 +74,42 @@
       end
     end
   end
+
+  describe '#execute :serve' do
+    describe '_site' do
+      data_dir = File.join(__dir__, '..', '..', '..', 'tests', 'full')
+      @thread;
+
+      before(:all) do
+        @thread = Thread.new do
+          jekyll_config_provider = Jekyll::PlantUml::JekyllConfigProvider.new(data_dir)
+          jekyll_config = jekyll_config_provider.provide('serve')
+          jekyll_commander = Jekyll::PlantUml::Commands::JekyllCommander.new(jekyll_config)
+          jekyll_commander.execute('serve').wait(Jekyll::Commands::Serve.mutex)
+        end
+        @thread.abort_on_exception = true
+    
+      end
+      
+      after(:each) do
+        capture_io do
+          Jekyll::Commands::Serve.shutdown
+        end
+  
+        Jekyll::Commands::Serve.mutex.synchronize do
+          if Jekyll::Commands::Serve.running?
+            Jekyll::Commands::Serve.run_cond.wait(Jekyll::Commands::Serve.mutex)
+          end
+        end
+      end
+
+      subject {
```

Comment:
Codacy found an issue: [Avoid using `{...}` for multi-line blocks.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5780130)